src/get-started/index.njk

```diff
@@ -0,0 +1,237 @@
+ </p>
+ 
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Prototype support
+ </h3>
+ 
+ <p class="govuk-body">
+ The GOV.UK Prototype Kit is used across governments and there is a significant amount of community support available if you get stuck or need help. 
+ </p>
+ 
+ <p class="govuk-body">
+ Inside HMRC you can reach out in our #community-prototype Slack channel or speak to members of HMRC’s Design Resources Team. Support is also available on the cross-government Slack in the #govuk-prototype-kit channel.
+ </p>
+ 
+ <p class="govuk-body">
+ You can also find out more in our prototyping club. We have a range of videos covering common tasks, like getting set up, routing and session data available on <a href="https://drive.google.com/drive/folders/1k9ZkMXE3WPpNsU126ekggSO5rWZ4MusK?usp=sharing/" class="govuk-link">Google Drive</a>.
+ </p>
+ 
+ <h3 class="govuk-heading-m" id="Hosting-and-sharing-prototypes">
+ Hosting and sharing prototypes
+ </h3>
+ 
+ <p class="govuk-body">
+ You will need a GitHub account, which you can create, and a digital HMRC login which can be set up for you by your team lead. Once you have set up your account you can use the 'automatic prototype creation tool'. 
+ </p>
+
+ <p class="govuk-body">
+ If you are working on a new service or a new prototype follow the guidance on <a href="https://confluence.tools.tax.service.gov.uk/display/PROT/Creating+a+new+prototype+repository/" class="govuk-link">Confluence</a> to create a new prototype repository. If you are working on a service where a prototype already exists, speak to the senior interaction designer on your programme or reach out on HMRC Slack <a href="https://join.slack.com/share/enQtNzY0NjE3NjY5NDE4MS1lNDczNTQxODk5YzE4MDk0MGY5YTBkY2YwMDdiNWNmZDRhMDdkOGIxMDg3YjA0ZTg5OWUwYjZhMTNjNjFlNjc3" class="govuk-link">#community-prototype</a>.
+ </p>
+ 
+ <p class="govuk-body">
+ You can use an existing GitHub account if you have one or you can create a new one. Full guidance on setting up your account and being added to the HMRC organisation is available in the <a href="https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/join-hmrc-on-github.html" class="govuk-link">MDTP handbook</a>.
+ </p> 
+ 
+ <p class="govuk-body">
+ Remember that a GitHub account is a public profile, linking you to HMRC.
+ </p>
+ 
+ <h2 class="govuk-heading-l" id="Other-tools-to-support-design-and-prototyping">
+ Other tools to support design and prototyping
+ </h2>
+ 
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Pattern Checker
+ </h3>
+ 
+ <p class="govuk-body">
+ Pattern Checker is an <a href="https://chromewebstore.google.com/detail/pattern-checker/amjjliajblignodfdjalnfkekkeflkph?hl=en-GB">automated pattern and accessibility checker</a> for government services that use GOV.UK and HMRC design patterns.
```

Comment:
Not everyone uses Chrome. Should we mentioned that this is a Chrome browser extension?

Reply:
Link needs `class='govuk-link'`.